README.md

```diff
@@ -1,2 +1,51 @@
-Hello-World
-In this project, you will set up a "Hello world" repository.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This project is my first ever on microverse and its a linter setup as well as a hello world
+
+![screenshot](./app_screenshot.png)
+
+This is a simple hello world project
+
+## Built With
+
+- HTML
+- CSS
+
+<!-- ## Live Demo
+
+[Live Demo Link](https://livedemo.com) -->
```

Comment:
- [ ] Please kindly uncomment and [ add a live demo](https://www.youtube.com/watch?v=laE_SeIab_0) of your project